Dockerfile

```diff
@@ -73,4 +66,14 @@ RUN chown -R spark_user:spark $CDM_SHARED_DIR
 # Switch back to non-root user
 USER spark_user
 
+# Set home directory for the spark user so that pipenv can install packages properly with spark user permissions
+ENV HOME=/opt/bitnami
+
+# install pipenv
+RUN pip3 install pipenv
+
+# install python dependencies
+COPY Pipfile* ./
+RUN pipenv sync --system
+
```

Comment:
With the pip installs this late the in file, they'll get reinstalled any time `src` changes.

Reply:
👍

Reply:
:+1: